src/main/resources/org/jenkins_ci/plugins/run_condition/core/StringsMatchCondition/config.properties

```
@@ -25,3 +25,4 @@
 arg1=String 1
 arg2=String 2
 ignoreCase=Case insensitive
+enviornmentVariables=enviornmentVariables
```

I think that the text should use spaces to separate words, since it is displayed to the user.
```suggestion
environmentVariables=Environment variables
```